docs/high-availability.md

```diff
@@ -71,6 +71,51 @@ http://<ip-addr>:9000
 
 Restart HAProxy to apply the configuration changes.
 
+### Example configuration: Citrix VPX Load Balancer
+
+**Note: This is extremely expermintial; so use at your own risk.**
+
+The config below shows how you can input this into the CLI on your Citrix VPX load balancer. You **must** replace **[IP ADDRESS]** with your network IPv4 address settings.
+
+#### Controller Entries
+```txt
+add server srv_kube_controller_01 [IP ADDRESS]
+add server srv_kube_controller_02 [IP ADDRESS]
```

Comment:
We want to have an odd number of kube_controllers. I understand this is an example but I worry people may find this and think we're somehow recommending 2 servers instead of 3. Could you please add a third srv_kube_controller?